README.md

```diff
@@ -61,6 +61,25 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Release guidelines
+
+Here are the recommended steps for both major and minor runtime releases.
+
+|Steps |Description |Major |Minor |
+|------|------------|------|------|
+|0 |Open an [issue](https://github.com/polkadot-fellows/runtimes/issues) for the release in the runtimes repo with the **version number**. | ✔️ | ✔️ |
+|1 |Identify **released crates**. | ✔️ | ❌ |
```

Comment:
Maybe call it "Update polkadot-sdk"? Otherwise "identitfy released crates" is quite generic.

Reply:
> Maybe call it "Update polkadot-sdk"? Otherwise "identitfy released crates" is quite generic.

Yes, the wording of these guidelines is generic on purpose.
Re: polkadot-sdk, since it is overseen by Parity only, I am not sure whether it makes sense to direct Fellowship members to do an "update".
"Identifiy released crates" is really just to say: check what is going on on the polkadot-sdk side of things, and follow up if an update is needed.

Reply:
> Re: polkadot-sdk, since it is overseen by Parity only, I am not sure whether it makes sense to direct Fellowship members to do an "update".

For sure not. However, the reality is that Parity "controls" the SDK that is used for building the runtimes. We probably also use some other crates, but nothing defines what the runtime is as the SDK.

Reply:
I have mentioned "update polkadot-sdk" in the doc.